themes/default/content/blog/azure-container-solutions/index.md

```diff
@@ -0,0 +1,181 @@
+
+### What we’d use it for
+
+The flexibility and extensibility offered by Kubernetes as a platform means it’s possible to run many types of workloads. Large companies have battled tested Kubernetes (and by extension, AKS) running everything from web applications, to machine learning.
+
+If you’re able to invest time into learning and understanding the ecosystem, and can dedicate people to maintaining and managing your Kubernetes implementation, AKS is a fantastic option for almost any workload you might consider.
+
+## Everything Else
+
+### Azure Red Hat OpenShift
+
+Azure RedHat OpenShift is an enterprise grade implementation of Kubernetes developed by Red Hat. OpenShift builds on Kubernetes by adding support, enterprise grade security and platform as a service integrations.
+
+If the idea of using Kubernetes for your container solutions seems appealing, but you need SLA’s around support and integrations, Azure’s Red Hat OpenShift solution is a great option.
+
+### Azure Service Fabric
+
+Azure Service Fabric is a container service focused on distributed applications, like microservice architectures or complex distributed compute environments.
+
+If you’re planning on running a complex architecture, Azure Service Fabric can be a great fit, but provides more orchestration capabilities than Azure App Service. Service Fabric can also extend outside Azure’s cloud into your on-premise data center if necessary.
+
+If your application has needs related to low latency, especially across distributed regions, Service Fabric is a great fit.
+
+### Azure Batch
+
+Azure Batch is a service focused on High Performance Compete (HPC) workloads. These workloads can often scale horizontally to thousands of processes, and can be difficult to orchestrate.
+
+Azure Batch makes this process simpler, by providing a managed service to handle the difficult processes.
+
+If you’re planning on passing large amounts of data to your application via a queuing mechanism and having that data processed very quickly, Azure Batch is definitely a service you should consider
+
+### Azure Functions
+
+Azure Functions is Azure’s function as a service implementation. It supports launching “function apps” inside containers as a deployment mechanism.
+
+Function as a Service products are often designed for short lived, event driven workloads. If you need to run a script to react to a worker queue or other event, Azure Functions are a great option. It’s important to remember that Functions are often billed based on how long they run, so short lived workloads are often better suited.
+
+## Wrap Up
+
+As you can see from this article, there are a number of options for running containers in Azure, and choosing the right one for your workload might be difficult. We’ve tried to give an overview of the current state of the world, but Microsoft continues to add options and capabilities to all these products.
```

Comment:
Generally, for accessibility, I try to avoid verbiage that refers to a sense or an action that may not be something someone can do (e.g., seeing something, clicking something).

```suggestion
There are a number of options for running containers in Azure, and choosing the right one for your workload might be difficult. We've tried to give an overview of the current state of the world, but Microsoft continues to add options and capabilities to all these products.
```